File: frontend-moviez-app/src/app/movie-multiplex-allotment/model/allotment.ts
export class Allotment {
    allotmentId: String;
    movieName: String;
    multiplexName: String;
    location: String;
    city: String;
    screenNumber: Number;

    constructor(allotmentId?: String,
        movieName?: String,
        multiplexName?: String,
        location?: String,
        city?: String,
        screenNumber?: Number) {
            
        this.allotmentId = allotmentId;
        this.movieName = movieName;
        this.multiplexName = multiplexName;
        this.location = location;
        this.city = city;
        this.screenNumber = screenNumber

    }
}

File: backend-moviez-app/search-service/src/main/java/com/application/search/model/Allotment.java
package com.application.search.model;

import lombok.AllArgsConstructor;
import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
@EqualsAndHashCode
@ToString
public class Allotment {
	private String allotmentId;
	private String movieName;
	private String multiplexName;
	private String city;
	private int screenNumber;
}
